### ISSUE TITLE
ERROR  : Requesting capability set 0x000001ffffffffff while permitted capability set is 0x0000003fffffffff

### ISSUE THREAD

**Issue by jpfeuffer** [singularityhub/singularity-docker#18]
Using your 3.11.0 image on a hosted GitHub Actions runner.

see https://github.com/OpenMS/OpenMS/actions/runs/4495601964/jobs/7909390460

**Comment by vsoch** [comment]
You shouldn't need to dockerize singularity for GitHub actions - try just installing it to the runner. https://github.com/singularityhub/install-singularity

**Comment by jpfeuffer** [comment]
Thank you! I will give it a try.

**Comment by jpfeuffer** [comment]
It worked by the way. Thanks.

**Comment by fcasson** [comment]
I have the same issue.  I need a docker image with singularity for gitlab CI tests on the docker runner executor

**Comment by fcasson** [comment]
I did run it in privileged mode, but maybe its a problem with my docker deamon env (ubuntu on WSL2, during "hello world" prep before gitlab GI)

**Comment by jpfeuffer** [comment]
Hi. Maybe my corresponding issue at singularity helps you:
https://github.com/sylabs/singularity/issues/1466

TLDR: Github runners do not have a recent enough daemon version.
But as always, GitHub is ignoring community feedback:
https://github.com/actions/runner-images/issues/7332
https://github.com/actions/runner-images/issues/7042

**Comment by fcasson** [comment]
It also looks like downgrading the singularity version might also help.  Will also try that...

**Comment by fcasson** [comment]
For posterity, here's a oneliner to reproduce the error (on some systems) on docker versions < 23

`docker run --privileged quay.io/singularity/singularity:v3.11.0 run docker://alpine`

**Comment by maawoo** [comment]
> EDIT2: From this links above, it also looks like downgrading the singularity version might also help, but since I have a solution by upgrading docker, I won't need to try that route.

Downgrading to v3.10 worked for me on Docker v24.0.2:
![image](https://github.com/singularityhub/singularity-docker/assets/56583917/168699b2-6654-4b66-a86e-f66c4405fecc)
